vcpkg/overlay/qgis/inverted.patch

```diff
@@ -1,30 +0,0 @@
-diff --git a/src/core/raster/qgshuesaturationfilter.cpp b/src/core/raster/qgshuesaturationfilter.cpp
```

Comment:
@nirvn am I correct that this is no longer needed with 3.28.0?

Reply:
@m-kuhn , correct